Pod/Classes/Playback/public/WistiaPlayer.swift

```diff
@@ -89,6 +89,14 @@ public protocol WistiaPlayerDelegate : class {
 
      */
     func wistiaPlayer(_ player:WistiaPlayer, willLoadVideoForMedia media:WistiaMedia, usingAsset asset:WistiaAsset?, usingHLSMasterIndexManifest: Bool)
+    
+    /**
+     Informs the delegate that the WistiaPlayer is attempting to determine a URL for loading a `WistiaAsset`, giving an opportunity
+     for the delegate to inject a url (such as a cached local asset url).
+     
```

Comment:
Let's add some more descriptive information here

`The `WistiaPlayer` will use the `URL` returned by this method for playback.  Return `nil` to allow the `WistiaPlayer` to determine the best playback URL.`